test/ut/sdk/test_compressor_torch.py

```diff
@@ -274,6 +274,55 @@ def test_torch_QAT_quantizer(self):
         assert math.isclose(model.relu.module.tracked_min_biased, 0.002, abs_tol=eps)
         assert math.isclose(model.relu.module.tracked_max_biased, 0.00998, abs_tol=eps)
 
+    def test_torch_quantizer_export(self):
+        config_list_qat = [{
+            'quant_types': ['weight'],
+            'quant_bits': 8,
+            'op_types': ['Conv2d', 'Linear']
+        }, {
+            'quant_types': ['output'],
+            'quant_bits': 8,
+            'quant_start_step': 0,
+            'op_types': ['ReLU']
+        }]
+        config_list_dorefa = [{
+            'quant_types': ['weight'],
+            'quant_bits': {
+                'weight': 8,
+            }, # you can just use `int` here because all `quan_types` share same bits length, see config for `ReLu6` below.
+            'op_types':['Conv2d', 'Linear']
+        }]
+        config_list_bnn = [{
+            'quant_types': ['weight'],
+            'quant_bits': 1,
+            'op_types': ['Conv2d', 'Linear']
+        }, {
+            'quant_types': ['output'],
+            'quant_bits': 1,
+            'op_types': ['ReLU']
+        }]
+        config_set = [config_list_qat, config_list_dorefa, config_list_bnn]
+        quantize_algorithm_set = [torch_quantizer.QAT_Quantizer, torch_quantizer.DoReFaQuantizer, torch_quantizer.BNNQuantizer]
+
+        for config, quantize_algorithm in zip(config_set, quantize_algorithm_set):
+            model = TorchModel()
+            model.relu = torch.nn.ReLU()
```

Comment:
what is this line used for?

Reply:
Follow the code of `test_torch_QAT_quantizer()` in ut. Change the op relu type to **ReLU** and it will match the type in config_list.